Description: remove generated bison parser files during clean
Origin: vendor
Bug-Debian: https://bugs.debian.org/652244
Forwarded: no
Author: gregor herrmann <gregoa@debian.org>
Last-Update: 2016-11-27

--- a/BASIC256.pro
+++ b/BASIC256.pro
@@ -161,3 +161,6 @@
 SOURCES 					+=	LEX/basicParse.tab.c 
 SOURCES 					+=	*.cpp
 
+# Generated Bison parser files
+QMAKE_CLEAN		+=	LEX/basicParse.tab.h
+QMAKE_CLEAN		+=	LEX/basicParse.tab.c
