Description: fix spelling error in binary
Forwarded: no
Author: Graham Inggs <ginggs@debian.org>
Last-Update: 2016-11-29
--- a/PreferencesWin.cpp
+++ b/PreferencesWin.cpp
@@ -105,7 +105,7 @@
 	//
 	// decimal digits slider
 	r++;
-	decdigslabel = new QLabel(tr("Digits of Precission to Display:"),this);
+	decdigslabel = new QLabel(tr("Digits of Precision to Display:"),this);
 	decdigsslider = new QSlider(Qt::Horizontal);
 	decdigsslider->setMinimum(SETTINGSDECDIGSMIN);
 	decdigsslider->setMaximum(SETTINGSDECDIGSMAX);
